build.sc

```diff
@@ -13,11 +13,11 @@ import $ivy.`com.github.lolgab::mill-mima::0.0.23`
 import de.tobiasroeser.mill.vcs.version.VcsVersion
 import com.github.lolgab.mill.mima._
 
-val scala3 = "3.3.1"
-val scala213 = "2.13.10"
-val scala212 = "2.12.17"
-val scalaJS1 = "1.12.0"
-val scalaNative04 = "0.5.0"
+val scala3 = "3.3.3"
+val scala213 = "2.13.14"
```

Comment:
it's now 2.13.14, and the nsc is updated https://github.com/scala/scala/pull/10406